### ISSUE TITLE
Replace API request timeout error message with something friendlier

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#614]
## Problem

Currently we display [this error message](https://github.com/WordPress/openverse-frontend/blob/ed649202b1ef4b0caf9710f8bec17bfb5a0df303/src/data/api-service.js#L38-L40) raw into the page, so if a search request times out it will just show "timeout of 30 seconds exceeded"

## Description

We should change this to something friendlier like "Whoops, it looks like that request took too long to complete. Please try again." (or something else that is nicer for a non-technical end-user).

cc @panchovm for help coming up with an appropriate message to share with users in this case.

## Alternatives

We could show a generic error message for errored out search requests.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by panchovm** [comment]
Here is an idea for this. I used your text proposal.

![imagen](https://user-images.githubusercontent.com/895819/150151033-6cebb396-f307-4f06-97f9-87a74f76c18c.png)

It is a basic layout that we can replicate in other cases. I have often seen these two other error messages and wondered if we should do the same. I hope we can solve this with the release, but my concern is visitors seeing 
it as often as the current version.

This template can be used in all those cases and changing the image and title. I chatted this with @zackkrida and he suggested tackling this after the release.

<img width="400" alt="CleanShot 2022-01-19 at 11 32 28@2x" src="https://user-images.githubusercontent.com/895819/150151337-412649fe-e31b-4d7c-82a2-3bddac9ce3c6.png">

**Comment by sarayourfriend** [comment]
I love it! I really dig using funny and relevant CC licensed images for our error pages 🙂

**Comment by dhruvkb** [comment]
I love the CC images. Could we use a consistent image + message layout for this that also applies to the 'No results page' in #602? I like the two columns side-by-side.

**Comment by panchovm** [comment]
You are right. We should use the #602 layout instead of designing more pages. On mobile we might keep the image + attribution otherwise it looks empty. I already updated the mockups in Releases file.

<img src="https://user-images.githubusercontent.com/895819/150394771-f6f00b46-d2b5-421b-82d9-0fc64e5f194a.png" width="200">

Here is attached the image compressed.

[Timeout - Waiting for a bite image.zip](https://github.com/WordPress/openverse-frontend/files/7907489/Timeout.-.Waiting.for.a.bite.image.zip)

**Comment by dhruvkb** [comment]
Updated PR #603 to include this error.

**Comment by obulat** [comment]
@dhruvkb , can we close this issue now that #603 is merged?

**Comment by dhruvkb** [comment]
#603 has implemented the components but it has not yet been used on the server error page. Currently, (the error layout)[https://github.com/WordPress/openverse-frontend/blob/ab53a6f8043d5d8425700fd48984dba16fe6b650/src/layouts/error.vue] shows 404 for _all errors!_.
